src/common/lib/polkadot-provider/query/genetic-analysts.js

```diff
@@ -0,0 +1,5 @@
+export async function queryGeneticAnalysts(api, accountId) {
+  let res = (await api.query.geneticAnalysts.geneticAnalysts(accountId))
```

Comment:
why are you using let instead of const and (? please update with better practice